libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccount.m

```diff
@@ -78,14 +78,6 @@ + (BOOL)supportsSecureCoding {
     return YES;
 }
 
-- (instancetype)init {
```

Comment:
Argh, Xcode now complains/warns about not overriding the designated initializer of the parent class.  One solution would be to just implement as pass-through:
```obj-c
- (instancetype)init {
    return [super init];
}
```
Exhaustively, you could leverage `NS_UNAVAILABLE` and throw an exception in the implementation, if we _really_ don't want people to go that route.